File: miscellaneous/sorting/bubble-sort/20230224/bubble-sort.php
<?php

declare(strict_types=1);

function bubbleSort(array $nums): array
{
    for ($i = 0; $i < count($nums) - 1; ++$i) {
        for ($j = 0; $j < count($nums) - $i - 1; ++$j) {
            $nums[$j] > $nums[$j+1] && ([$nums[$j], $nums[$j+1]] = [$nums[$j+1], $nums[$j]]);
        }
    }

    return $nums;
}

File: miscellaneous/sorting/bubble-sort/20230224/bubble_sort.py
from typing import List;


def bubble_sort(nums: List[int]) -> List[int]:
    for i in range(len(nums) - 1):
        for j in range(len(nums) - 1 - i):
            if nums[j] > nums[j+1]:
                [nums[j], nums[j+1]] = [nums[j+1], nums[j]]

    return nums
